ibis/tests/util.py

```diff
@@ -43,13 +43,13 @@ def wrapper(backend, *args, **kwargs):
     return wrapper
 
 
-def skipif_backend(skip_backend):
+def skipif_backend(*backend_classes):
     def wrapped(f):
         @functools.wraps(f)
         def wrapper(backend, *args, **kwargs):
-            if isinstance(backend, skip_backend):
-                pytest.skip('Skipping {} test'.format(str(backend)))
-            else:
-                return f(backend, *args, **kwargs)
+            for backend_class in backend_classes:
```

Comment:
You don't need to loop here since isinstance accepts a tuple of types as the second argument.